addons/l10n_ar/models/account_move.py

```diff
@@ -34,6 +34,17 @@ def _l10n_ar_get_document_number_parts(self, document_number, document_type_code
     l10n_ar_afip_service_start = fields.Date(string='AFIP Service Start Date', readonly=True, states={'draft': [('readonly', False)]})
     l10n_ar_afip_service_end = fields.Date(string='AFIP Service End Date', readonly=True, states={'draft': [('readonly', False)]})
 
+    def _is_manual_document_number(self):
+        res = super()._is_manual_document_number()
+        # NOTE: There is a corner case where 2 sales documents can have the same number for the same DOC from a different vendor,
+        # in that case, the user can create a new Sales Liquido Producto Journal
+
+        # When issuer is supplie the numbering works opposite (supplier enumerate the invoices, and the customer encode the bill)
+        if self.l10n_latam_use_documents and ((self.journal_id.type == 'sale' and not self.journal_id.l10n_ar_is_pos) or (
+           self.journal_id.type == 'purchase' and self.journal_id.l10n_ar_is_pos)):
+            return not res
```

Comment:
This 'not res' is just confusing...  Why not put the good logic here?

Reply:
@zaoral besides addressing this comment, can you close this PR and open a new one to see if the ci/codeowner issue get resolved?
Thanks!

Reply:
Hi @jco-odoo and @vbe-odoo 

I hope you are doing well.
I have updated this part of the code to `return True.`  directly True. because if any of those two cases encoded in the if are true then we should use manual numbering.

You will see this one impacting the new pr I will create in a moment.
Best Regards